templates/api/knowledge-panels/product/product_tags_panels.tt.json

```diff
@@ -0,0 +1,15 @@
+[% FOREACH panel IN panel.taxonomy_panels %]
+    {
```

Comment:
Each panel contains elements, and those elements can be themselves panels, but when an element is a panel, it just lists the panel id.

You can look at the generated panels for a product to see how the panel elements are defined:
https://fr.openfoodfacts.org/api/v3/produit/7622210449283/prince-gout-chocolat-lu?fields=knowledge_panels

e.g.

```
{
elements: [
{
element_type: "panel",
panel_element: {
panel_id: "environmental_score"
}
},
```